tasks/create.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Create UFM container
+  docker_container:
+    exposed_ports: "{{ item.value.exposed_ports | default(omit) }}"
+    image: "{{ item.value.image }}"
+    name: "{{ item.value.container_name }}"
+    network_mode: "{{ item.value.network_mode | default(omit) }}"
+    privileged: "{{ item.value.privileged | default(omit) }}"
+    published_ports: "{{ item.value.published_ports | default(omit) }}"
+    read_only: "{{ item.value.read_only | default(omit) }}"
+    restart_policy: "{{ mlnx_ufm_restart_policy }}"
+    restart_retries: "{{ mlnx_ufm_restart_retries if mlnx_ufm_restart_policy == 'on-failure' else omit }}"
+    state: "{{ item.value.enabled | ternary('present', 'absent') }}"
+    tmpfs: "{{ item.value.tmpfs | default(omit) }}"
+    volumes: "{{ item.value.volumes }}"
+  notify: Restart {{ item.value.container_name }}
```

Comment:
Why would we restart after creating?

Reply:
I'm assuming that `restart: true` on a stopped container is the same as starting it. Is this not the case?

Reply:
or does `state: present` start the container as well?

Reply:
I can confirm `state: present`, puts it in the created state and doesn't start it:
```
[will@juno ~]$ docker ps -a
CONTAINER ID        IMAGE                                              COMMAND                  CREATED             STATUS                    PORTS               NAMES
ae81ccc5b88d        ubuntu:latest                                      "/bin/bash"              8 seconds ago       Created                                       ubuntu-delele-me
```

Reply:
this was my test:
```
[will@juno ~]$ cat /tmp/test4.yml
---

- hosts: localhost
  tasks:
    - docker_container:
        state: present
        name: ubuntu-delele-me
        image: ubuntu:latest
    - docker_container:
        state: started
        restart: true
        name: ubuntu-delele-me
```

Reply:
Should we use state: started here instead then?

Reply:
But I was trying to avoid the container being started and then restarted. The restart handler from the config task only gets invoked once, right?

Reply:
ok, I see.